packages/set/src/index.ts

```diff
@@ -18,74 +18,81 @@ const $KEYS = Symbol("track-keys");
 export class ReactiveSet<T> extends Set<T> {
   #triggers = new TriggerCache<T | typeof $KEYS>();
 
-  constructor(values?: Iterable<T> | null) {
+  constructor(values?: readonly T[] | null) {
```

Comment:
The specification of Set accepts an Iterable<T>. Why change this to a const array?

Reply:
Great catch! I was probably referring to ES6 types that do not support Iterables. Will fix it :)